tests/test_runner.sh

```diff
@@ -13,4 +13,4 @@ function execute() {
 }
 
 # Run the tests.
-execute time bats -j $(nproc) --tap "${@:-.}"
+execute time bats --tap "${@:-.}"
```

Comment:
this is as well, you are disabling parallel testing for integration tests

Reply:
if you add `-T` to bats it will add `in xxx ms` after each test name to the log.
Then you can check these timings to see if you spot any particular slow test that could be optimized.